unittests/UnitTestFramework.xml

```diff
@@ -34,7 +34,7 @@ When developing tests for a certain feature the developer has to:
 			  If any assertion is not fulfilled then an error will be "thrown" (like an exception).
 		- You can override setUp() and tearDown() methods if required
 		 (setup will run before any test method, teardown will run after all have completed)
-		- You can skip a test by using the skipTest method (using the G_Assert global variable).
+		- You can skip a test by invoking the skipTest method (using the G_Assert global variable).

```

Comment:
No need to change now. I'd recommend we add () to make methods/procedures clear (and help with doc generation, which can then cross link). E.g. skipTest()